src/components/userInfo/UserInfo.js

```diff
@@ -1,3 +1,18 @@
+/*
+
+
+ECS:
+
+Overall this component seem overly complex especially around state and how the profile image is being
+stored to send over to the API. The mulitple references to the picture in state and the code logic 
+around maintining that make this code hard to follow.
+
+I would also spend some time understaind promises or the alternate syntax of async/await
+
```

Comment:
I would agree that the code was hard to follow as well, but it worked from the state it was written, so I didn't want to break it by changing how it was done. I think it would've been best to have the update picture it's own separate component imported into the UserInfo.js

Reply:
Yeah, a lot of that mess is my fault. I should have taken the time to clean it up after me and shanita got it working.